<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Install Unreal Engine 4</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="githubLabel">
     <property name="text">
      <string>Create a Github account</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="stepsInfoLabel">
     <property name="text">
      <string>You have to do this steps first before installing:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="linkLabel">
     <property name="text">
      <string>Link your Github account to your Epic Games account</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="linkButton">
     <property name="text">
      <string>Link</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QPushButton" name="installButton">
     <property name="text">
      <string>Install</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="githubButton">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="epicLabel">
     <property name="text">
      <string>Create an Epic Games account</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="epicButton">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Activate 2-factor-authentication</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="activateButton">
     <property name="text">
      <string>Activate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
